x-pack/plugins/ml/public/embeddables/single_metric_viewer/embeddable_single_metric_viewer_container.tsx

```diff
@@ -159,6 +173,24 @@ export const EmbeddableSingleMetricViewerContainer: FC<
 
   const containerPadding = 10;
 
+  if (error) {
+    return (
+      <EuiCallOut
+        title={
+          <FormattedMessage
+            id="xpack.ml.singleMetricViewerEmbeddable.errorMessage"
+            defaultMessage="Unable to load the ML single metric viewer data"
```

Comment:
Nit - perhaps `Unable to load the data for the single metric viewer`.

Could you change the text on the other two panels to have a similar format?

Reply:
Updated in cc1826fa25be3a34295f5621989ae35ca7d6dd60